cookbook/sequential_drop_banks.rst

```diff
@@ -0,0 +1,273 @@
+target to be "lit" in order for the sequence to start.
+
+* Rule 3: When the sequence resets, advance the first target from "off" to "lit"
+
+When a shot is in the "off" state and gets hit, we want to fire the reset coil
+for the target so that the target stays up.
+
+* Rule 4: When an "off" shot is hit, reset its coil
+
+We can apply all of these rules based on the corresponding events, like follows.
+
+
+.. code-block:: mpf-config
+
+  ##! mode: sequential_drops
+
+  event_player:
+    # When the mode starts, reset the drop sequence
+    mode_sequential_drops_started: reset_drop_sequence
+
+    # When the sequence is completed, reset the drop sequence
+    logicblock_drop_sequence_complete: reset_drop_sequence
+
+    # When the sequence resets, advance the first target
+    reset_drop_sequence: advance_drop_1
+
+    # When an "off" shot is hit, reset its coil
+    drop_1_off_hit: reset_drop_1
+    drop_2_off_hit: reset_drop_2
+    drop_3_off_hit: reset_drop_3
+    drop_4_off_hit: reset_drop_4
+
+The above configuration requires that each drop target coil has the
+corresponding reset events, as configured below.
+
+.. code-block:: mpf-config
+
+  #! switches:
+  #!   s_drop_1:
+  #!     number: 1
+  #!   s_drop_2:
+  #!     number: 2
+  #!   s_drop_3
+  #!     number: 3
+  #!   s_drop_4:
+  #!     number: 4
+  #! coils:
+  #!   c_drop_1:
+  #!     number: 1
+  #!   c_drop_2:
+  #!     number: 2
+  #!   c_drop_3
+  #!     number: 3
+  #!   c_drop_4:
+  #!     number: 4
+  ##! machine config.yaml
+
+  drop_targets:
+    drop_1:
+      switch: s_drop_1
+      reset_coil: c_drop_1
+      reset_events: ball_starting, machine_reset_phase_3, reset_drop_1
+    drop_2:
+      switch: s_drop_1
+      reset_coil: c_drop_1
+      reset_events: ball_starting, machine_reset_phase_3, reset_drop_2
+    drop_3:
+      switch: s_drop_1
+      reset_coil: c_drop_1
+      reset_events: ball_starting, machine_reset_phase_3, reset_drop_3
+    drop_4:
+      switch: s_drop_1
+      reset_coil: c_drop_1
+      reset_events: ball_starting, machine_reset_phase_3, reset_drop_4
+
+
+Step 5. Rewards for progression and completion
+----------------------------------------------
+
+When a drop target is hit, The sequence logic block keeps track of whether it is
+the part of the sequence or not. We can easily award points for progression with
+the *logicblock_(name)_hit* event (when a lit target is hit) and the
+*logicblock_(name)_complete* event (when the full sequence is completed).
+
+
+.. code-block:: mpf-config
+
+  ##! mode: sequential_drops
+
+  variable_player:
+    logicblock_drop_sequence_hit:
+      score: 1000
+    logicblock_drop_sequence_complete:
+      score: 50_000
+
+
+Full Example Code
+-----------------
+
+The full code from this example can be found as a fully-working game template in
+the MPF Examples repository.
+
+https://github.com/missionpinball/mpf-examples/tree/dev/cookbook/sequential_drop_banks
+
```

Comment:
Link looks dead. We could also include the full example here with a test. What do you think?

Reply:
Link is just dead until https://github.com/missionpinball/mpf-examples/pull/16 merges